Older kernels do not have trace/define_trace.h [1] and if there
is something there that we can backport we will define it
in or compat headers.

[1] http://lxr.linux.no/#linux+v2.6.31/include/trace/define_trace.h

--- a/drivers/net/wireless/iwlwifi/iwl-devtrace.h
+++ b/drivers/net/wireless/iwlwifi/iwl-devtrace.h
@@ -263,4 +263,6 @@ TRACE_EVENT(iwlwifi_dev_ucode_event,
 #define TRACE_INCLUDE_PATH .
 #undef TRACE_INCLUDE_FILE
 #define TRACE_INCLUDE_FILE iwl-devtrace
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,30))
 #include <trace/define_trace.h>
+#endif
--- a/net/mac80211/driver-trace.h
+++ b/net/mac80211/driver-trace.h
@@ -1,7 +1,9 @@
 #if !defined(__MAC80211_DRIVER_TRACE) || defined(TRACE_HEADER_MULTI_READ)
 #define __MAC80211_DRIVER_TRACE
 
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,27))
 #include <linux/tracepoint.h>
+#endif /* (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,27)) */
 #include <net/mac80211.h>
 #include "ieee80211_i.h"
 
@@ -772,4 +774,6 @@ TRACE_EVENT(drv_flush,
 #define TRACE_INCLUDE_PATH .
 #undef TRACE_INCLUDE_FILE
 #define TRACE_INCLUDE_FILE driver-trace
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,30))
 #include <trace/define_trace.h>
+#endif /* (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,30)) */
